qe_metrics/libs/jira.py

```diff
@@ -17,9 +17,21 @@ def __init__(self, config_file: str) -> None:
             config_file (str): Path to the yaml file holding database and Jira configuration.
         """
         self.logger = get_logger(name=self.__class__.__module__)
-
         self.jira_config = parse_config(config_file)["jira"]
 
+    def __enter__(self) -> "Jira":
+        return self
```

Comment:
`__enter__` should call `self.connection`

In context manager, I expect to have "working" object while I'm in the context, meaning I have a connection once I'm inside the context.

Same for `Database` class